spring-integration-stomp/src/main/java/org/springframework/integration/stomp/outbound/StompMessageHandler.java

```diff
@@ -199,6 +199,11 @@ public void handleFrame(StompHeaders headers, Object payload) {
 			}
 		}
 
+		@Override
+		public void handleTransportError(StompSession session, Throwable exception) {
```

Comment:
The `handleException()` is called only once from the `DefaultStompSession`:
```
	/**
	 * Handle a message received from the remote host.
	 * @param message the message
	 */
	void handleMessage(Message<P> message);
```
But that is for the incoming part and it is addressed in the `StompInboundChannelAdapter`.

Here in the `StompMessageHandler` we handle:
```
else if (StompCommand.ERROR.equals(command)) {
	invokeHandler(this.sessionHandler, message, stompHeaders);
}
```
in the `handleFrame()` above. That's is how the `DefaultStompSession` is implemented.
Although I see your point and any `handleFrame()` may result with an exception which will be caught by the `handleException()`. So, I'm really going to add the similar log implementation there as well.
Will address it together with the #1430 new concerns.